### DISCUSSION TITLE
Atlas windows 10 to 11 activation key

### DISCUSSION THREAD

**Discussion by 0veRReacT** [Atlas-OS/Atlas#1220] [Help & Support]
"Hello everyone! I have a quick question for you. I switched from the version of Atlas for Windows 10 to the one for Windows 11, and I wanted to know if the automatic activation of Windows will work or if I need to get a new key ?

**Comment by Ast3risk-ops** [comment]
Your current windows 10 key should work for 11.

**Reply by 0veRReacT** [reply]
So I should wait for windows to auto activate?

**Reply by Ast3risk-ops** [reply]
> So I should wait for windows to auto update?

Reinstall to 11 with an ISO (since you have atlas, you have to) and use the same product key or microsoft account you used for 10.

**Reply by 0veRReacT** [reply]
Problem is that I can't switch from local account to Microsoft account since windows is not activated. And I think I will not find the key...

**Reply by CletusThaGod** [reply]
My issue is similar to what @0veRReacT is describing. it wont let me sign into my MS account.

**Comment by 0veRReacT** [comment]
I actually get error code 0xC004F213 windows didn't detect any product key

**Reply by bandit8623** [reply]
> I actually get error code 0xC004F213 windows didn't detect any product key

you likely used a diff version then.  pro vs home?

**Reply by 0veRReacT** [reply]
I have a pro licence and I'm pretty sure i chose same version as previous : win 10 pro to win 11 pro.
If I activated my license with a win pro N should it work with a non N win pro ? Cause I chose a non N version here.

**Reply by antoinepirlot** [reply]
I installed Atlas and now Windows has been deactivated after some weeks.

**Reply by antoinepirlot** [reply]
> I installed Atlas and now Windows has been deactivated after some weeks.

I solved the problem, it was an issue with Microsoft

**Reply by CletusThaGod** [reply]
I believe this is a related issue here... I've installed atlas correctly, followed install doc to the letter, windows pro and all. no product key sticker for this PC, which usually is fixed with a digital key in my microsoft account. But I cant Sign in to my microsoft account at all. the only way i can find that lets me even try to sign in, lets me enter email and then authorize the login using my phone, and then says "oops something happened. BLAH BLAH BLA", and usually it would ask for my password and the product key activates. how can i solve this? is it an issue with atlas or Microsoft? or is it intentional for whatever reason they may have? any help stepping through this would be great. Im very familiar with installing linux and windows and other less popular OS's. But this one has me feeling like a noob again.

thanks

**Reply by bandit8623** [reply]
> I believe this is a related issue here... I've installed atlas correctly, followed install doc to the letter, windows pro and all. no product key sticker for this PC, which usually is fixed with a digital key in my microsoft account. But I cant Sign in to my microsoft account at all. the only way i can find that lets me even try to sign in, lets me enter email and then authorize the login using my phone, and then says "oops something happened. BLAH BLAH BLA", and usually it would ask for my password and the product key activates. how can i solve this? is it an issue with atlas or Microsoft? or is it intentional for whatever reason they may have? any help stepping through this would be great. Im very familiar with installing linux and windows and other less popular OS's. But this one has me feeling like a noob again.
> 
> thanks
you are trying to use your key tied to microsoft account?  no need a key.  after activated you can login to your account.  at least thats what i did.  i didnt use the one tied tol my account.  take a look at g2a

**Reply by CletusThaGod** [reply]
yes I am, that's the only key I have available for pro and when I sign in to my MS account, I get the "oops" message and I cant do anything from there but close the window

**Reply by bandit8623** [reply]
> yes I am, that's the only key I have available for pro and when I sign in to my MS account, I get the "oops" message and I cant do anything from there but close the window

I guess I don't know then.  I used a diff version of 11 than I had on my account. So I just grabbed a cheap key from g2a.  Used that key and have been good. I was able to login to the Microsoft store and such. My version was enterprise though

**Reply by antoinepirlot** [reply]
> > I installed Atlas and now Windows has been deactivated after some weeks.
> 
> I solved the problem, it was an issue with Microsoft

I reinstalled Windows and try to activate my key without Atlas OS and it didn't work (I had exactly the same issue) I reinstalled Atlas OS and tried again and still didn't work. So I decided to buy a new key (as my key was 5 years old, I thought it had expired), then I insert the key in Windows (with Atlas OS) then, the activation worked. 

I don't know what really happened in my case and if it was related to Atlas OS or just Microsoft.

**Reply by CletusThaGod** [reply]
im gonna go wuth what homiew above suggested using G2A for a cheap key and just run with that i guess. i would assume this is a problem that needs some attention at least, am i wrong?

**Comment by bandit8623** [comment]
What is did u use? There are a few sources. Same exact version?
